apps/asap-server/src/handlers/webhooks/webhook-calendar-created.ts

```diff
@@ -0,0 +1,175 @@
+import AWS from 'aws-sdk';
+import Joi from '@hapi/joi';
+import { auth, JWTInput, JWT } from 'google-auth-library';
+import { framework as lambda } from '@asap-hub/services-common';
+import { WebhookPayload, Calendar } from '@asap-hub/squidex';
+import {
+  region,
+  googleApiCredentialsSecretId,
+  googleApiUrl,
+  asapApiUrl,
+} from '../../config';
+import { http } from '../../utils/instrumented-framework';
+
+import { Handler } from '../../utils/types';
+import validateRequest from '../../utils/validate-squidex-request';
+import Calendars, { CalendarController } from '../../controllers/calendars';
+
+export const webhookCalendarCreatedHandlerFactory = (
+  subscribe: SubscribeToEventChanges,
+  unsubscribe: UnsubscribeFromEventChanges,
+  calendarController: CalendarController,
+): Handler =>
+  http(
+    async (request: lambda.Request): Promise<lambda.Response> => {
+      await validateRequest(request);
+
+      const bodySchema = Joi.object({
+        type: Joi.string().required(),
+        payload: Joi.object({
+          data: Joi.object().required(),
+        })
+          .unknown()
+          .required(),
+      })
+        .unknown()
+        .required();
+
+      const { payload, type: event } = lambda.validate(
+        'body',
+        request.payload,
+        bodySchema,
+      ) as WebhookPayload<Calendar>;
+
+      if (event === 'CalendarsUpdated') {
+        if (
+          !payload.dataOld ||
+          !payload.dataOld.id ||
+          payload.dataOld.id.iv === payload.data.id.iv
+        ) {
+          return {
+            statusCode: 200,
+            payload: payload.data.id,
+          };
+        }
+
+        if (payload.dataOld.resourceId) {
+          try {
+            await unsubscribe(payload.dataOld.resourceId?.iv, payload.id);
+          } catch (error) {
+            // eslint-disable-next-line no-console
+            console.error(error);
+          }
```

Comment:
```suggestion
         await unsubscribe(payload.dataOld.resourceId?.iv, payload.id).catch( e => console.error(e))
```

This is, most likely, not properly formatted, but the point is, since it is a one-liner, we can make it smaller and maintain the same readability

Reply:
oh i wouldnt not want to mix .catch with async/await. I really think it's much better to use one or the other